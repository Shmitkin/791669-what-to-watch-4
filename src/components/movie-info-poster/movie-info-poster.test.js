import React from "react";
import renderer from "react-test-renderer";
import MovieInfoPoster from "./movie-info-poster.jsx";

it(`Should MovieInfoPoster render correctly`, () => {
  const tree = renderer
    .create(
        <MovieInfoPoster
          altDesc = {`description`}
          image = {`src/image`}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
