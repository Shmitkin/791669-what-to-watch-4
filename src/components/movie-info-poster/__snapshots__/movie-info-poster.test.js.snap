// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieInfoPoster render correctly 1`] = `
<div
  className="movie-card__poster movie-card__poster--big"
>
  <img
    alt="Pulp Fiction poster"
    height="327"
    src="img/pulp-fiction.jpg"
    width="218"
  />
</div>
`;
